core/app/assets/styles/mixins/_screen-reader.scss

```diff
@@ -0,0 +1,34 @@
+// Only display content to screen readers
+//
+// See: https://a11yproject.com/posts/how-to-hide-content/
+// See: https://hugogiraudel.com/2016/10/13/css-hide-and-seek/
+
+@mixin sr-only() {
```

Comment:
There is already "sr" handling in this pr

Reply:
Remove duplication

Reply:
done